siobrultech_protocols/gem/const.py

```diff
@@ -0,0 +1,6 @@
+_ESCAPE_SEQUENCE = "^^^"
+_SYSTEM_PREFIX = _ESCAPE_SEQUENCE + "SYS"
+_REQUEST_PREFIX = _ESCAPE_SEQUENCE + "RQS"
+
+CMD_DELAY_NEXT_PACKET = _SYSTEM_PREFIX + "PDL"
+CMD_GET_SERIAL_NUMBER = _REQUEST_PREFIX + "SRN"
```

Comment:
For the benefit of typing, do you think we should make these an `enum` instead?

Reply:
The one benefit that I can see (and should have elaborated on), is that the `formatter` of `ApiCall` would then return this `enum` type, as oppose to an arbitrary string.  I haven't put a lot of thought into other commands yet, so that might be the incorrect way to think about this.

Aside: it might make sense to have a `Typevar` for the `Callable`s for `formatter` and `parser`

Reply:
Yeah, that's what I was thinking of as well. Some commands take arguments which just get added on to the command string, so the formatter would still have to return a string.